src/test/java/com/cm39/cm39/user/mapper/UserMapperTest.java

```diff
@@ -0,0 +1,113 @@
+package com.cm39.cm39.user.mapper;
+
+import com.cm39.cm39.user.domain.UserDto;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@DisplayName("회원 mapper 테스트")
+//@ExtendWith(SpringExtension.class)
+@SpringBootTest
+class UserMapperTest {
+
+    @Autowired
+    private UserMapper userMapper;
+
+    // 모든 테스트 실행 전 회원 전체 삭제
+    @BeforeEach
+    void deleteAllUser() {
+        userMapper.deleteAllUser();
+        assertEquals(countUser(), 0);
+    }
+
+    @DisplayName("회원Id로 조회")
+    @Test
+    void selectUserByUserId() {
+        // insert user
+        UserDto userDto = getUserDto();
+        userMapper.insertUser(userDto);
+
+        List<UserDto> allUserList = userMapper.selectAllUser();
+        assertNotNull(allUserList);
+        assertTrue(allUserList.size() > 0);
+
+        // select user by userId
+        UserDto selectedUser = userMapper.selectUserByUserId(userDto.getUserId());
+        assertNotNull(selectedUser);
+        assertEquals(userDto.getUserId(), selectedUser.getUserId());
+        assertEquals(userDto.getPwd(), selectedUser.getPwd());
+        assertEquals(userDto.getGrdId(), selectedUser.getGrdId());
+        assertEquals(userDto.getUserName(), selectedUser.getUserName());
+        assertEquals(userDto.getBirth(), selectedUser.getBirth());
+        assertEquals(userDto.getGndr(), selectedUser.getGndr());
+        assertEquals(userDto.getTelNo(), selectedUser.getTelNo());
+        assertEquals(userDto.getUserStatCode(), selectedUser.getUserStatCode());
+        assertEquals(userDto.getSnsTypeCode(), selectedUser.getSnsTypeCode());
+        assertEquals(userDto.getRegId(), selectedUser.getRegId());
+        assertEquals(userDto.getUpId(), selectedUser.getUpId());
```

Comment:
equals를 오버라이딩해서 비교해보는 건 어떨까요?
```suggestion
assertEquals(userDto, selectedUser);
```